### ISSUE TITLE
Add a dark theme or relax CSP

### ISSUE THREAD

**Issue by WereCatf** [pypa/warehouse#10391]
**What's the problem this feature will solve?**

The website might no longer burn its users' eyes out.

**Describe the solution you'd like**

Add a dark theme -- using the CSS media feature "_prefers-color-scheme_"? -- or relax CSP, so one can use one of the many extensions that allow one to use custom styles on websites.

**Comment by WereCatf** [comment]
Yeah, I expected that no one would like the idea of relaxing the CSP. There other option, however, has gained traction slowly over the years with a website here, website there adopting it and it seems like it'd be the way to go.

**Comment by rajpratyush** [comment]
is this issue open for work?

**Comment by vitaly-zdanevich** [comment]
I created the userstyle for it https://gitlab.com/vitaly-zdanevich-styles/pypi

<img width="1200" height="1920" alt="Image" src="https://github.com/user-attachments/assets/6d796e9c-15a1-423d-a4f6-2bb5b41b1d0e" />
